chore(android): Clarify how to disable Timber integration (#6213)
getsentry/sentry-docs: src/platforms/android/configuration/integrations/timber.mdx

However, you can still override the default behaviour by adding your own instance of the `SentryTimberIntegration`. For that, refer to the [manual installation](/platforms/android/configuration/integrations/timber/#configure) section below.

### Disable

> If you would like to disable the Timber integration (e.g. to not exceed your event quota), but keep using other auto-installation features, remove the `sentry-android-timber` dependency from the app configurations in `app/build.gradle`:

Reviewer: ```suggestion
If you want to disable the Timber integration (to not exceed your event quota, for example), but keep using other auto-installation features, remove the `sentry-android-timber` dependency from the app configurations in `app/build.gradle`:
```
